lib/speech/index.js

```diff
@@ -0,0 +1,903 @@
+        var result = chunk.results[0];
+        if (result.alternatives && result.alternatives.length) {
+          return done(null, result.alternatives);
+        }
+      }
+      return done(null);
+    }
+  );
+
+  // The GRPC stream
+  var recognizeStream = this.streamingRecognize({
+    streamingConfig: reqOpts
+  });
+
+  // The simplified user stream
+  var detectStream = duplexify.obj(
+    toRecognizeRequestStream,
+    toSpeechRecognitionAlternativesStream
+  );
+
+  // Forward metadata to the user stream
+  recognizeStream.on('response', function(chunk) {
+    detectStream.emit('response', chunk);
+  });
+
+  // Transform user data into RecognizeRequest objects
+  toRecognizeRequestStream.pipe(recognizeStream);
+
+  // Transform recognition results to a simpler form
+  recognizeStream.pipe(toSpeechRecognitionAlternativesStream);
+
+  return detectStream;
+};
+
+/**
+ * Create an Operation object. See {module:speech/createRecognizeJob} to create
+ * an operation.
+ *
+ * @throws {Error} If a name is not provided.
+ *
+ * @param {string} name - The name of the operation.
+ * @return {module:speech/operation}
+ *
+ * @example
+ * var operation = speech.operation('68850831366825');
+ *
+ * operation.get(function(err) {});
+ */
+Speech.prototype.operation = function(options, decoder) {
+  if (is.string(options)) {
+    options = {
+      name: options
+    };
+  }
+  if (!options || !options.name) {
+    throw new Error('A name must be specified for an operation.');
+  }
+
+  return new Operation(this, options, decoder);
+};
+
+/**
+ * Gets the latest state of a long-running operation. Clients can use this
+ * method to poll the operation result at intervals as recommended by the
+ * API service.
+ *
+ * @resource [Operations: GetOperation API Documentation]{@link https://cloud.google.com/speech/reference/rpc/google.longrunning#operations}
+ * @resource [Operations: Operation API Documentation]{@link https://cloud.google.com/speech/reference/rpc/google.longrunning#operation}
+ *
+ * @param {string|module:speech/operation} id - The operation id or operation
+ *     instance.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:speech/operation} callback.operation - `Operation` object. See
+ *     [`Operation`](https://cloud.google.com/speech/reference/rpc/google.longrunning#operation).
+ * @param {object} callback.apiResponse - Raw API response.
+ *
+ * @example
+ * //-
+ * // <h3>Provide operation id</h3>
+ * //-
+ * speech.getOperation('68850831366825', function(err, operation, apiResponse) {
+ *   console.log(operation);
+ * });
+ *
+ * //-
+ * // <h3>Provide operation instance</h3>
+ * //-
+ * var operation = speech.operation('68850831366825');
+ * speech.getOperation(operation, function(err, operation, apiResponse) {
+ *   console.log(operation);
+ * });
+ */
+Speech.prototype.getOperation = function(name, callback) {
```

Comment:
I might be wrong, but I think this goes against our general object hierarchy. Usually to retrieve a single object, we would use something like

```js
speech.operation('my-op').get(function() {});
```

And we would reserve `get` prefixed methods for retrieving a collection of objects.